src/cli/serve/__tests__/read_yaml_config.js

```diff
@@ -1,15 +1,14 @@
 import expect from 'expect.js';
 import { join, relative, resolve } from 'path';
 import readYamlConfig from '../read_yaml_config';
-import sinon from 'auto-release-sinon';
 
 function fixture(name) {
   return resolve(__dirname, 'fixtures', name);
 }
 
 describe('cli/serve/read_yaml_config', function () {
   it('reads a single config file', function () {
-    const config = readYamlConfig(fixture('one.yml'));
+    readYamlConfig(fixture('one.yml'));
```

Comment:
Do we need this call anymore? Are there other places where we have situations like this?